recipe/meta.yaml

```
@@ -12,16 +12,16 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
-    - python >=3.6
-    - pbr
+    - python
+    - pbr ==5.6.0
```

```suggestion
    - pbr
```
We usually don't strictly pin python build tools as it makes maintenance more difficult without providing much benefits. (contrary to pinning stuff like C libraries).